luxonis_ml/data/loader.py

```diff
@@ -91,6 +91,10 @@ def __init__(
                 synced_classes = json.load(file)
             for task in synced_classes:
                 self.dataset.set_classes(classes=synced_classes[task], task=task)
+            skeletons_file = os.path.join(self.dataset.metadata_path, "skeletons.json")
+            with open(skeletons_file) as file:
```

Comment:
Same comment here. I guess this will just fail if there is not skeletons.json file?

Can this affect training for other non-skeleton tasks?

Reply:
Yes, it seems like it will impact other trainings. Maybe it would be wise to check if the file exists and, if so, call the get_skeletons() function?

Reply:
@tersekmatija Could you please check this fix?